src/config/constants/ifo.ts

```diff
@@ -32,7 +32,7 @@ const ifos: Ifo[] = [
     campaignId: '511100000',
     articleUrl: 'https://pancakeswap.medium.com/hot-cross-hotcross-ifo-to-be-hosted-on-pancakeswap-10e70f1f6841',
     tokenOfferingPrice: 0.05,
-    isV1: false,
+    version: 2,
```

Comment:
Using `version` is more scalable in my opinion. Undoubtedly there will be a version 3+ as we grow.